mmv1/third_party/terraform/services/container/node_config.go.tmpl

```diff
@@ -806,6 +806,14 @@ func schemaNodeConfig() *schema.Schema {
 						ForceNew: true,
 						Description: `If enabled boot disks are configured with confidential mode.`,
 				},
+				"local_ssd_encryption_mode": {
+					Type:             schema.TypeString,
+					Optional:         true,
+					Computed:         true,
```

Comment:
This means that the field has a default value returned by the server even when unset, and won't be returned as `null`. Is that correct? 

`Computed` means that if the field is removed from the config we will preserve the old value instead of revert to null/empty/falsey. If it will only ever be returned if set explicitly, `Computed` can be removed, which is generally preferable. However, if the API may change the default or return non-null defaults in the future (which is breaking for an API but tends to happen regardless) we should keep it.

Reply:
This field should only be used (or configured) in boot disk so there shouldn't be any further changes. Will remove `computed`